### DISCUSSION TITLE
Announcement that "old" actions will be removed from the Marketplace

### DISCUSSION THREAD

**Discussion by mpdude** [community/community#25016] [Actions]
<p>I received a message from the GitHub Actions Product Team by email:</p>
<blockquote>
<p>Thanks for using GitHub Actions during the preview, and for creating an action and publishing it to the GitHub Marketplace!</p>
</blockquote>
<blockquote>
<p>During the preview period – after you created your action – we added a configuration file for actions that is required for publishing to the Marketplace. Since this configuration is required, we will be removing all actions that do not have it from the Marketplace on March 1, 2020.</p>
</blockquote>
<blockquote>
<p>If you want your action to remain in the Marketplace, you’ll need to make the following changes:</p>
</blockquote>
<blockquote>
<ol>
<li>Add an action.yml to your repository that describes your action.</li>
<li>Update your repository’s README to include instructions for using your action in the new YAML workflow configuration language, instead of the outdated HCL language.</li>
<li>Publish a new version of the action that includes both the action.yml and changes to the README.</li>
</ol>
</blockquote>
<blockquote>
<p>You can find more information about the action.yml at: <a href="https://help.github.com/en/actions/automating-your-workflow-with-github-actions/metadata-syntax-for-github-actions" rel="nofollow noopener">https://help.github.com/en/actions/automating-your-workflow-with-github-actions/metadata-syntax-for-github-actions</a></p>
</blockquote>
<blockquote>
<p>If you do not update your action it will be removed from the Marketplace on March 1, 2020.</p>
</blockquote>
<blockquote>
<p> </p>
</blockquote>
<p>My Action (<a href="https://github.com/webfactory/ssh-agent/" rel="nofollow noopener">https://github.com/webfactory/ssh-agent/</a>) seems to include all this.</p>
<p>Did anyone else get this announcement so that it was maybe not precisely targeted? Am I missing something?</p>
<p>Thanks!</p>

**Comment by ethomson** [comment, marked as answer]
<p>Hi mpdude -</p>
<p>Apologies for not including more information in that message.  It is targeted to people who have published marketplace actions and do not have an action.yml.  Although webfactory/ssh-agent *was* published with an action.yml, but you have other actions in the marketplace that lack it.  In this case: <a href="https://github.com/marketplace/actions/webfactory-composer-require-checker" rel="nofollow noopener">https://github.com/marketplace/actions/webfactory-composer-require-checker</a></p>
<p>You’ll need to either remove that from the marketplace or update it with an action.yml and publish a new version.</p>
<p>Cheers-</p>
<p>-ed</p>

**Comment by nick-invision** [comment]
<p>This was also confusing to me as my primary actions all are configured properly.  I forgot that I published a hello-world action during the preview.  In my case, I just searched for my name in the Marketplace and found the hello-world that could be delisted.</p>

**Comment by mpdude** [comment]
<p>D’oh! I forgot we published that one.</p>
